daisy_integration_tests/scripts/run_daisy_and_check_logs.sh

```diff
@@ -50,7 +50,7 @@ run_daisy() {
 
 verify_logs() {
   echo 'BuildStatus: Waiting for logs to propagate to StackDriver.'
-  sleep 30
+  sleep 120
```

Comment:
Do we know why is this necessary? What changed yesterday that made this change necessary?

Reply:
The timeline of this lines up: https://status.cloud.google.com/incident/google-stackdriver/19004 . That said, I like this longer period since it benefits the case where we don't expect logs to be uploaded.